routes.js

```diff
@@ -1,58 +1,76 @@
-const { error } = require('console');
 const express = require('express');
-const router = express.Router();
+const createMyCustomApi = express.Router();
+const {loadMembers, saveMembers} = require('./dataAccess')
 
-// Middleware
-router.use(express.json());
+// Endpoint to get a list of all team members (get request)
+createMyCustomApi.get('/getAllMembers', function(req, res) {
+    const members = loadMembers();
+    res.status(200).json(members);
+})
 
-// Get method (Get all member)
-router.get('/getAllMembers', (req, res) => {
-  res.send(teamMembersArray);
-  if(!error) {
-      res.status(200);
-  }
+// Endpoint to add a new member to the team (post request)
+createMyCustomApi.post('/addNewMember', (req, res) => {
+    const newMember = {
+        memberId: req.body.memberId,
+        role: req.body.role,
+        firstname: req.body.firstname,
+        lastname: req.body.lastname
+    }
+    if (!newMember.memberId || !newMember.role || !newMember.firstname || !newMember.lastname) {
+        res.status(400).json({ message: 'All fields are required...' });
+        return
+    }
+    const members = loadMembers();
+    if(members.some(member => member.memberId === newMember.memberId)) {
+        res.status(409).json({ message: 'This record ID already exists...' });
+        return
+    }
+    members.push(newMember);
+    saveMembers(members);
+    res.status(200).json({ message: 'OK' });
 });
 
-// Post method (Add a new member)
-router.post('/addNewMember', (req, res) => {
-  const member = {
-    memberId: req.body.memberId,
-    role: req.body.role,
-    firstname: req.body.firstname,
-    lastname: req.body.lastname
-  }
-  teamMembersArray.push(member);
-  res.send(member);
-  if(!error) {
-    res.status(200);
-  }
+// Endpoint to update the information about an existing team member (put request)
+createMyCustomApi.put('/updateExistMember/:memberId', (req, res) => {
+    const memberToUpdateId = parseInt(req.params.memberId, 10);
```

Comment:
So-called `magic numbers` are a bad practice, because anyone reading the code in couple of weeks lacks the context of what they mean. In that case, in order to understand what `10` means, a reader has to break the flow and google syntax of `parseInt` function.
As a rule of thumb - extract magic values into explanatory variables. Then name of variable going to give a context to the reader.
In this case you can extract a global single variable for entire `routes.js`
```
const base10 = 10;
```
So this and similar lines read like:
```
const memberToUpdateId = parseInt(req.params.memberId, base10);
```

Reply:
Done!